cmd/opm/serve/serve.go

```diff
@@ -59,12 +73,37 @@ will not be reflected in the served content.
 	}
 
 	cmd.Flags().BoolVar(&s.debug, "debug", false, "enable debug logging")
-	cmd.Flags().StringVarP(&s.port, "port", "p", "50051", "port number to serve on")
 	cmd.Flags().StringVarP(&s.terminationLog, "termination-log", "t", "/dev/termination-log", "path to a container termination log file")
+	cmd.Flags().StringVarP(&s.port, "port", "p", "50051", "port number to serve on")
+	cmd.Flags().BoolVar(&s.pprofEnabledHttp, "pprof-enable-endpoint", false, "enable pprof HTTP endpoint")
+	cmd.Flags().BoolVar(&s.pprofEnabledFile, "pprof-enable-file", false, "enable pprof profile file writing to /tmp/serve.[cpu|mem].prof")
 	return cmd
 }
 
 func (s *serve) run(ctx context.Context) error {
+
+	if s.pprofEnabledHttp {
+		// pprof tool HTTP endpoint
+		go func() {
+			s.logger.Infof("starting pprof endpoint at %s:%s", "localhost", defaultProfPort)
+			http.ListenAndServe("localhost:"+defaultProfPort, nil)
```

Comment:
This won't be queryable in a cluster environment unless we run a sidecar proxy that binds on something other than `localhost`. Is that intentional?

Reply:
Ignorance on my part as what the desired configuration should be.  When I'm operating in an unknown area (to me), I tend to try to force the errors to be early & loud, so they're not missed.  
Should this be just `:port` instead?